src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/http_client.html.twig

```diff
@@ -8,6 +8,17 @@
             <span class="sf-toolbar-value">{{ collector.requestCount }}</span>
         {% endset %}
 
+        {% set text %}
+            <div class="sf-toolbar-info-piece">
+                <b>Total requests</b>
+                <span>{{ collector.requestCount }}</span>
+            </div>
+            <div class="sf-toolbar-info-piece">
+                <b>HTTP errors</b>
+                <span class="sf-toolbar-status sf-toolbar-status-red">{{ collector.errorCount }}</span>
+            </div>
+        {% endset %}
```

Comment:
can you please share a screenshot showing the effect of this?

Reply:
Oh, OK already here:
![image](https://user-images.githubusercontent.com/243674/73676852-66721980-46b5-11ea-8cfc-e88d20909438.png)